features/diagnostics/DetailViewContent.feature

```diff
@@ -17,12 +19,26 @@ Feature: Diagnostics: Detail view content
     Then the "$details" element contains ""environment": "universal""
 
   Scenario: Reads code search value from URL
-    When I visit the "/diagnostics?codeSearch=(groups%257Cusers)" URL
+    When I visit the "/diagnostics?codeSearch=(groups%257Cusers)&codeRegExp=true&codeFilter=true" URL
 
     Then the "$code-block-search-input" element contains "(groups|users)"
+    And the "$code-block-regexp-mode-button[aria-pressed='true']" element exists
+    And the "$code-block-filter-mode-button[aria-pressed='true']" element exists
 
   Scenario: Stores code search value in URL
     When I visit the "/diagnostics" URL
-    And I "input" "(groups|users)" into the "$code-block-search-input" element
+
+    Then the "$code-block-regexp-mode-button[aria-pressed='true']" element doesn't exist
+    And the "$code-block-filter-mode-button[aria-pressed='true']" element doesn't exist
+
+    When I "input" "(groups|users)" into the "$code-block-search-input" element
 
     Then the URL contains "/diagnostics?codeSearch=(groups%257Cusers)"
+
+    When I click the "$code-block-regexp-mode-button" element
+
+    Then the URL contains "/diagnostics?codeSearch=(groups%257Cusers)&codeRegExp=true"
+
+    When I click the "$code-block-filter-mode-button" element
+
+    Then the URL contains "/diagnostics?codeSearch=(groups%257Cusers)&codeRegExp=true&codeFilter=true"
```

Comment:
I saw that you made some changes below with the boolean things like we discussed earlier, doesn't this mean we don't want to have `true` in the URLs now?

Reply:
Yeah, if you know how to do this with `router.push`, we can change this, but I’m not actually sure that’s "supported"? Best I could get was `codeFilter=` by submitting `{ codeFilter: '' }`.